Tests/Worker/WorkerTests.cs

```diff
@@ -47,7 +47,7 @@ public async System.Threading.Tasks.Task TestWorkflowAsyncExecution()
             var workflow = GetConductorWorkflow();
             ApiExtensions.GetWorkflowExecutor().RegisterWorkflow(workflow, true);
             var workflowIdList = await StartWorkflows(workflow, quantity: 15);
-            await ExecuteWorkflowTasks(workflowCompletionTimeout: TimeSpan.FromSeconds(20));
+            await ExecuteWorkflowTasks(workflowCompletionTimeout: TimeSpan.FromSeconds(40));
```

Comment:
We need to review this later but for the moment increasing this and other sleep/timeouts to prevent test abortion due to:

```
The active test run was aborted. Reason: Test host process crashed : Unhandled exception. System.OperationCanceledException: The operation was canceled
```